README.md

```diff
@@ -93,12 +93,8 @@ To run tests, run the following command: Not Applicable
 
 website is deployed using Github page:
 here is the link
-https://atifzada.github.io/Portfolio-Project/
+https://atifzada.github.io/Capstone-Project-01/
```

Comment:
- [ ] I kindly suggest that you ensure that your links in your deployed website are working because this is one of the requirements of this projects. Kindly see a video demo below:


https://user-images.githubusercontent.com/52934128/225667903-1b8e1167-c13b-4b76-b2bc-eafe8375c681.mp4